verification/curator-service/ui/README.md

```diff
@@ -1,4 +1,30 @@
-## Available Scripts
+# Global Health Curator Portal UI
+
+This directory contains the code for the UI of the Global Health Curator Portal accessible here:
+
+- [dev](https://dev-curator.ghdsi.org)
```

Comment:
perhaps worth documenting the default port for local here as well

Reply:
good call, done